tamr_client/categorization/project.py

```diff
@@ -0,0 +1,52 @@
+from tamr_client._types import (
+    CategorizationProject,
+    Dataset,
+    Instance,
+    JsonDict,
+    Session,
+    URL,
+)
+from tamr_client.dataset import _dataset, unified
+
+
+def _from_json(url: URL, data: JsonDict) -> CategorizationProject:
+    """Make Categorization project from JSON data (deserialize)
+
+    Args:
+        url: Project URL
+        data: Project JSON data from Tamr server
+    """
+    return CategorizationProject(
+        url, name=data["name"], description=data.get("description")
+    )
+
+
+def manual_labels(
+    session: Session, instance: Instance, project: CategorizationProject
+) -> Dataset:
+    """
+    Get manual labels from a Categorization project
+    Args:
+        instance: Tamr instance containing this dataset
+        project: Tamr project of this Unified Dataset
+
+    Returns:
+        Dataset containing manual labels
+
+    """
+    unified_dataset = unified.from_project(
+        session=session, instance=instance, project=project
+    )
+    labels_dataset_name = unified_dataset.name + "_manual_categorizations"
+    dataset_url = URL(
+        instance=instance,
+        path=session.get(
+            url=str(
+                URL(
+                    instance=instance,
+                    path=f"datasets?filter=name=={labels_dataset_name}",
+                )
+            )
+        ).json()[0]["relativeId"],
```

Comment:
Couple points:
- [ ] Defensively check if filtering results in 0 or many datasets
- [ ] Use `dataclasses.replace` to create new variants of URL
- [ ] Split up code into multiple lines for clarity

```suggestion
    from dataclasses import replace

    instance_url = URL(instance)
    matches = session
        .get(str(instance_url) + "datasets?filter=name=={labels_dataset_name}")
        .json()
    
    if len(matches) == 0:
        raise NotFound(...)
    if len(matches) > 1:
        raise Ambiguous(...)
        
    dataset_path = matches[0]["relativeId"]
    dataset_url = replace(instance_url, path=dataset_path)
```

Reply:
added these changes to the script and tested against a TR instance. seems to be all good